crates/oxc_linter/src/snapshots/anchor_is_valid.snap

```diff
@@ -0,0 +1,89 @@
+---
+source: crates/oxc_linter/src/tester.rs
+assertion_line: 119
+expression: anchor_is_valid
+---
+  ⚠ eslint-plugin-jsx-a11y(anchor-is-valid): Missing `href` attribute for the `a` element.
+   ╭─[anchor_is_valid.tsx:1:1]
+ 1 │ <a />
+   ·  ─
+   ╰────
+  help: Provide an href for the `a` element.
+
+  ⚠ eslint-plugin-jsx-a11y(anchor-is-valid): Use an incorrect href for the 'a' element.
+   ╭─[anchor_is_valid.tsx:1:1]
+ 1 │ <a href={undefined} />
+   ·  ─
+   ╰────
+  help: Provide a correct href for the `a` element.
+
+  ⚠ eslint-plugin-jsx-a11y(anchor-is-valid): Use an incorrect href for the 'a' element.
+   ╭─[anchor_is_valid.tsx:1:1]
+ 1 │ <a href={null} />
+   ·  ─
+   ╰────
+  help: Provide a correct href for the `a` element.
+
+  × Unterminated string
+   ╭─[anchor_is_valid.tsx:1:1]
+ 1 │ <a href=' />;
+   ·         ─────
```

Comment:
Is this a broken test or indented?

Reply:
I had noticed this,  I found that OXC generates this AST in this case. So, I thought it should not hit this rule🧐
```
{
  "type": "Program",
  "start": 0,
  "end": 0,
  "sourceType": {
    "language": {
      "typeScript": {
        "isDefinitionFile": false
      }
    },
    "moduleKind": "module",
    "variant": "jsx",
    "alwaysStrict": false
  },
  "directives": [],
  "hashbang": null,
  "body": []
}
```

Reply:
So is this intended, we need a comment explaining it if this is the case.

Reply:
This is not intentional, so have we commented out this case and made remarks?

Reply:
I just checked the source, this is actually intended to be parse error.